plugins/inputs/riemann_listener/riemann_listener.go

```diff
@@ -17,13 +17,13 @@ import (
 	"time"
 
 	riemanngo "github.com/riemann/riemann-go-client"
-	riemangoProto "github.com/riemann/riemann-go-client/proto"
+	riemango_proto "github.com/riemann/riemann-go-client/proto"
```

Comment:
What about `riemann` and `riemann_proto`? We are already in go anyway 😉

Reply:
Good point, changed.